openfl/interface/workspace.py

```diff
@@ -109,7 +110,12 @@ def create(prefix, template):
 
 
 @workspace.command(name='export')
-def export_():
+@option('-o', '--pip-install-options', required=False,
+        type=str, multiple=True, default=tuple,
+        help='Options for remote pip install. '
+        'You may pass several options in quotation marks alongside with arguments, '
```

Comment:
Can you align text to text (Second and third row of help text under first row, not under argument `help`)?

Reply:
done

Reply:
If you add only tab, then it will be that error, but if you align text to text ('You ...' strictly under 'Options...' ). Like this https://github.com/intel/openfl/blob/2d66cd816a78670d6a99ef04669f5fa47bcf45f7/openfl/interface/collaborator.py#L255